hyperspy/docstrings/signal.py

```diff
@@ -140,6 +140,22 @@
            number of bins is capped by this number to avoid a MemoryError
            being raised by :func:`numpy.histogram`."""
 
+
+HISTOGRAM_RANGE_ARGS = """range : (float, float), optional
```

Comment:
Docstring could be reused in `get_histogram` and `plot_histogram`, but there the parameter is called `range_bins`